service/service_construction.go

```diff
@@ -111,6 +111,8 @@ func (s ConstructionService) ConstructionMetadata(
 			if input.Metadata != nil {
 				if input.Metadata.UnwrapBridgeTx {
 					gasLimit, err = s.getBridgeUnwrapTransferGasLimit(ctx, input.From, input.Value, input.Currency)
+				} else {
+					return nil, wrapError(errInvalidInput, "UnwrapBridgeTx must be populated if input.Metadata is provided")
 				}
```

Comment:
I think this reads better:

```suggestion
				if !input.Metadata.UnwrapBridgeTx {
					return nil, wrapError(errInvalidInput, "UnwrapBridgeTx must be populated if input.Metadata is provided")
				}
				
				gasLimit, err = s.getBridgeUnwrapTransferGasLimit(ctx, input.From, input.Value, input.Currency)
```

Reply:
Agreed...was too confined to fitting this within the code.

Reply:
4580e42